packages/admin/admin/src/form/FinalFormToggleButtonGroup.tsx

```diff
@@ -1,27 +1,39 @@
-import { ButtonBase } from "@mui/material";
+import { ButtonBase, Typography, type TypographyProps } from "@mui/material";
 import { css, styled } from "@mui/material/styles";
 import { type ReactNode } from "react";
 import { type FieldRenderProps } from "react-final-form";
 
-interface Props<FieldValue> extends FieldRenderProps<FieldValue, HTMLDivElement> {
-    options: Array<{ value: FieldValue; icon: ReactNode }>;
+export interface FinalFormToggleButtonGroupProps<FieldValue> extends FieldRenderProps<FieldValue, HTMLDivElement> {
+    options: Array<{ value: FieldValue; label: ReactNode }>;
```

Comment:
We should describe this rename if we keep the old component.

Reply:
add an information to the migration guide, that the options signature have changes from `icon` to `label`